src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1029,6 +1029,9 @@ class AddItemsPage extends Component {
     this.fetchAddItemsPageData();
     if (!this.props.isPaginated) {
       this.fetchLineItems();
+    } else if (this.state.isFirstPageLoaded) {
+      // refetching update items when first page s already loaded
```

Comment:
// Workaround for refetching items from scrach when the first page was already loaded and table is paginated